### PR TITLE
added-isDisabled-to-TextLink-testkit-and-docs

### PR DESCRIPTION
**Pull request by shypeleg**
## Before you request a review, please make sure to check the following boxes:

- [ ] Explain what has changed.
- [ ] Explain why the change is needed.
- [ ] Added tests.
- [ ] Update documentation.
- [ ] Merge master with no conflicts.
- [ ] Make sure the build is passing on ci.
- [ ] If this pr closes any issue, please add a github link to close it.
- [ ] If you changed something in the product design/UX, you got the approval of [Ben Benhorin](https://wix.slack.com/messages/@benb).

### CODE REVIEW

File: src/BaseComponents/TextLink/TextLink.driver.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ const textLinkDriverFactory = ({element, wrapper, component}) => {
     getLink: () => element.href,
     getTarget: () => element.target,
     getRel: () => element.rel,
+    isDisabled: () => component.props.disabled,
```
Comment (shlomitc) [inline src/BaseComponents/TextLink/TextLink.driver.js]: should check the actual element and not the props

**Comment by shlomitc** [comment]
Thanks for the PR, please fix the requested change.

**Comment by lbelinsk** [comment]
@shypeleg thanks for the contribution :)
If it is possible, in addition to Shlomi's comment, if it is not too much to ask, we prefer to check the driver's method by using them in some test, so if it is possible to add a small test which uses this new method, it will be awesome! If not, we will do it

**Comment by shypeleg** [comment]
אני אשתדל למצוא לזה זמן בסופש :  ]

**Comment by argshook** [comment]
:) תודה לך על העבודה שלך! אבל אנגלית בבקשה ,לא כולם מדברים עברית פה

**Comment by shypeleg** [comment]
Oh sorry I was just answering an email from a friend and didn't realize its
a github thingy :O  : ]

**Comment by lbelinsk** [comment]
keep-alive ping

**Comment by shypeleg** [comment]
no need we decided a disabled link is ... a (formatted) text.

**Pull request closed without merging**